client/client-multi/client-java-core/src/test/java/org/kaaproject/kaa/client/context/FlexibleExecutorContextTest.java

```diff
@@ -0,0 +1,185 @@
+package org.kaaproject.kaa.client.context;
```

Comment:
missed license in this file
see documentation:
[(https://docs.google.com/document/d/1gesi7pElMnoT5ocefV0qLooEMnZuTvTc81xSdJQGGTI)]